README.md

```diff
@@ -120,6 +120,31 @@ The host container is the one we will connect, it will show a folder icon when y
 If you connect to the DevContainer successfully, you will see your project folders.
 ![](https://i.imgur.com/3bzRwnN.png)
 
+### Set up AWS SQS Service for worker in Codepraise-api.
+
+When we attempt to run CodePraise, we need to replace certain tokens in the secret.yml file. One of the most critical setups is the SQS service, as it directly affects the usage of workers.
+
+1. Login [AWS](https://signin.aws.amazon.com/signin?redirect_uri=https%3A%2F%2Fconsole.aws.amazon.com%2Fconsole%2Fhome%3FhashArgs%3D%2523%26isauthcode%3Dtrue%26state%3DhashArgsFromTB_ap-southeast-2_f0f961ef2b42dff4&client_id=arn%3Aaws%3Asignin%3A%3A%3Aconsole%2Fcanvas&forceMobileApp=0&code_challenge=koB4iW2c_PuWNp6eO1klqErpW-HGUIYN8kAnMIMF4sA&code_challenge_method=SHA-256) and choose "IAM user", if you don't have an account, create a new one.
+![](https://i.imgur.com/kKvsWMh.jpg)
```

Comment:
Please discuss with @taylor-wu96 how to change these imgur links to local jpg or png files in a `docs/images` folder.

Reply:
ok got it~